content/legal/responsible_disclosure.md

```diff
@@ -51,17 +49,29 @@ Only the following services are in-scope:
 - Social engineering attacks (e.g. phishing)
 - Email
 
-**The following types of bugs do not qualify for bounties:**
+#### Common False Positives
+**The following types of issues are commonly reported and do not qualify for bounties:**
 - CSRF on forms that are available to anonymous users (e.g., signup, login, contact, Intercom)
-- Self-XSS and issues exploitable only through Self-XSS
+- Self-XSS and issues exploitable only through self-XSS
 - Clickjacking and issues only exploitable through clickjacking
+- Email authentication settings
+- Email rate limiting/flood attacks
 - Functional, UI and UX bugs and spelling mistakes
 - Descriptive error messages (e.g. stack traces, application or server errors)
 - HTTP 404 codes/pages or other HTTP error codes/pages
 - Banner disclosure on common/public services
 - Disclosure of known public files or directories, (e.g. robots.txt)
 - Presence of application or web browser "autocomplete" or "save password" permission
 
+#### Example Issues
+The following types of issues are examples of reports we have paid cash bounties for:
+- Arbitrary directory traversal on Git endpoints
+- Improper direct object references
+- HTML injection
+
+#### Submissions
+Please email [security@aptible.com](mailto:security@aptible.com) to report an issue. If you would like to include a video, please link to an unpublished YouTube URL. We cannot accept binary videos or files.
```

Comment:
I would also ask that people don't attach PDFs. The same problems are present there as with videos.